packages/ruleset/src/functions/operation-summary-exists.js

```diff
@@ -0,0 +1,33 @@
+const { LoggerFactory } = require('../utils');
+
+let ruleId;
+let logger;
+
+module.exports = function(operation, _opts, context) {
+  if (!logger) {
+    ruleId = context.rule.name;
+    logger = LoggerFactory.newInstance().getLogger(ruleId);
+  }
+  return operationSummary(operation, context.path);
+};
+
+function operationSummary(operation, path) {
+  logger.debug(`${ruleId}: checking summary for operation at location: ${path.join('.')}`);
```

Comment:
According to the build log:
```suggestion
  logger.debug(
    `${ruleId}: checking summary for operation at location: ${path.join('.')}`
  );
```

Reply:
oops, forgot to run the linter before I opened the PR :(
it's fixed now